dashboards-observability/public/components/explorer/docTable/detailTable/docDetailTable.tsx

```diff
@@ -31,22 +24,24 @@ export interface DocViewRenderProps {
   columns?: string[];
   filter?: DocViewFilterFn;
   hit: any;
-  // indexPattern: any;
   onAddColumn?: (columnName: string) => void;
   onRemoveColumn?: (columnName: string) => void;
 }
 
+const fieldType = 'string';
```

Comment:
These props are just required to use DocViewTableRow which I borrowed from discover. For now since our data table for each doc row is just a static table, but later when we are adding more features to support the similar functionalities in discover to click to sort/add filter/etc..then these will change accordingly.